kraft/utils/op.py

```diff
@@ -32,10 +33,22 @@
 from __future__ import unicode_literals
 
 import os
+import shlex
```

Comment:
Is this new dependency used?

Reply:
It is integrated to python core. It is not a 3rd party dependency.

Reply:
:+1: